simple/ice_model.F90

```diff
@@ -524,8 +526,12 @@ subroutine ice_model_init ( Ice, Time_Init, Time, &
   !enddo
   !enddo
 
+  ! check if io domain set up, if so check if file can be opened
+  do_io = associated( mpp_get_io_domain(Ice%domain) )
```

Comment:
I had figured if it wasn't using the io domain it would be better not to set it, but probably doesn't make much of a difference. I'll change it to that instead.